src/helper/Reply.h

```diff
@@ -73,6 +73,11 @@ class Reply
     MemBuf other_;
 };
 
```

Comment:
Please also check the adjusted PR description: Does it still accurately reflect this PR intent?